contract/src/main.rs

```diff
@@ -347,14 +364,18 @@ pub extern "C" fn init() {
         .unwrap_or_revert_with(NFTCoreError::FailedToCreateDictionary);
     storage::new_dictionary(PAGE_TABLE)
         .unwrap_or_revert_with(NFTCoreError::FailedToCreateDictionary);
+    storage::new_dictionary(EVENTS).unwrap_or_revert_with(NFTCoreError::FailedToCreateDictionary);
+    storage::new_dictionary(EVENT_ID_TRACKER)
+        .unwrap_or_revert_with(NFTCoreError::FailedToCreateDictionary);
```

Comment:
```suggestion
```

as per previous comment, does not seem to be used else where

Reply:
Applied suggestion